net/mwan3/files/etc/hotplug.d/iface/15-mwan3

```diff
@@ -7,18 +7,17 @@
 . /lib/mwan3/common.sh
```

Comment:
Up a little bit, there's an include to functions.sh. I would remove and replace with /etc/rc.common in the shebang line.

Reply:
Why include /etc/rc.common in the shebang line? It seems like it is for init scripts, and not hotplug scripts. It does include the procd.sh but that is just for the procd_running and procd_send_signal functions.

Reply:
Ah ignore that.